docs/SMCWG-charter.md

```diff
@@ -77,21 +77,27 @@ capitalized terms in this Charter.
 The authorized scope of the SMCWG SHALL be to discuss, adopt, and maintain
 policies, frameworks, and sets of standards related to the issuance and
 management of S/MIME certificates by third-party CAs under a publicly trusted
-root, limited as follows:
+root.
 
-- Baseline verification of control over email addresses, including those used
-  by a natural person or a legal entity, or used by automated systems such as
-  for mailing lists
-- Identity validation for natural persons and legal entities
+The primary deliverable shall be scoped as follows:
+
+- Verification of control over email addresses
 - Key management and certificate lifecycle (subject to coordination with other
   Forum CWGs to ensure consistency and avoid redundancy)
 - Certificate profiles for S/MIME certificates and Issuing CA certificates
   (including the appropriateness of extensions and when those extensions should
   be present)
 - CA operational practices, physical/logical security, etc.
 
-The SMCWG SHALL exercise caution to ensure that its work product does not impede
-the issuance of certificates with other EKU types.
+In addition to these primary deliverables, the SMCWG MAY also address:
+- Identity validation for natural persons and legal entities, in the context of
+  S/MIME certificates.
+- Certificate profiles for S/MIME certificates to convey information related to
+  natural person and legal identity information.
+
+Deliverables of the SMCWG SHALL be constrained to those certificates
+technically capable of issuing certificates with the emailProtection
+(OID: 1.3.6.1.5.5.7.3.4) Extended Key Usage (EKU).
```

Comment:
"certificates technically capable of issuing certificates with the ... EKU, or that contain the EKU?"

Obviously, the end-entity certificates themselves need to be in scope too, and I believe your language precludes that.

Reply:
Hopefully resolved in https://github.com/sleevi/cabforum-docs/pull/9/commits/1d5669864fdb18d0b34d380cebf37255a72a3786 - if that language works for you, could you click "resolve conversation" so I know?

Reply:
Yes, thank you, that resolves my issue.  Where is "resolve conversation"?  I don't see it.